### DISCUSSION TITLE
Advantages of osiset

### DISCUSSION THREAD

**Discussion by hexican0x** [osiset/laravel-shopify#1180] [Q&A]
Hiya! I'm about to build a Shopify app for the first time. Came across this library and the official Shopify PHP API library (https://github.com/Shopify/shopify-api-php) and template (https://github.com/Shopify/shopify-app-template-php) that works with the CLI. Wondering what are the advantages of using the osiset app, over the official versions?

**Comment by moonpetaldev** [comment]
There are various advantages and disadvantages, but one clear advantage this package has over Shopify's is that it actually provides documentation, whereas Shopify does not. (plus Shopify have historically put PHP users at a disadvantage and always put Node/Ruby users first)

**Comment by Kyon147** [comment]
Hey @hexican0x 

I also did a quick pro and con here https://github.com/osiset/laravel-shopify/discussions/1211
